Zero-Gravity-Store-React/src/Pages/CartWidget/Components/CartProducts.js

```diff
@@ -0,0 +1,102 @@
+        <>
+        <div className="horizontal-card flex">
+            <div className="card-header">
+              <img
+                className="cart_img"
+                src="https://cdn1.epicgames.com/offer/77f2b98e2cef40c8a7437518bf420e47/93d4e401-0a3d-4c03-9867-13ad384708b2_1200x1600-b26d34f747ae0f6721b5f75232c462fe?h=854&resize=1&w=640"
+                alt="load"
+              />
+              <span className="badge flex">New Release</span>
+            </div>
+            <div className="card-text">
+              <span className="game-tag">Base Game</span>
+              <h1>Cyperpunk 2077</h1>
+              <p>
+                Cyberpunk 2077 is an open-world, action-adventure story set in
+                Night City, a megalopolis obsessed with power, glamour and body
+                modification
+              </p>
+              <div>
+                <div className="price flex">
+                  <span className="tag">-50%</span><span className="line">₹2,999</span
+                  ><span className="bold">₹1,499.50</span>
+                </div>
+              </div>
+              <div><small>Sale ends 2/10/2022 at 9:30 PM</small></div>
+              <div className="action-btn">
+                <button className="btn btn-solid">
+                  <i className="fa fa-plus-circle"></i>Move to wishlist
+                </button>
+                <button className="btn btn-solid">Remove</button>
+              </div>
+            </div>
+          </div>
+          <div className="horizontal-card flex">
+            <div className="card-header">
+              <img
+                className="cart_img"
+                src="https://cdn1.epicgames.com/offer/77f2b98e2cef40c8a7437518bf420e47/93d4e401-0a3d-4c03-9867-13ad384708b2_1200x1600-b26d34f747ae0f6721b5f75232c462fe?h=854&resize=1&w=640"
+                alt="load"
+              />
+              <span className="badge flex">New Release</span>
+            </div>
+            <div className="card-text">
+              <span className="game-tag">Base Game</span>
+              <h1>Cyperpunk 2077</h1>
+              <p>
+                Cyberpunk 2077 is an open-world, action-adventure story set in
+                Night City, a megalopolis obsessed with power, glamour and body
+                modification
+              </p>
+              <div>
+                <div className="price flex">
+                  <span className="tag">-50%</span><span className="line">₹2,999</span
+                  ><span className="bold">₹1,499.50</span>
+                </div>
+              </div>
+              <div><small>Sale ends 2/10/2022 at 9:30 PM</small></div>
+              <div className="action-btn">
+                <button className="btn btn-solid">
+                  <i className="fa fa-plus-circle"></i>Move to wishlist
+                </button>
+                <button className="btn btn-solid">Remove</button>
+              </div>
+            </div>
+          </div>
```

Comment:
Have a card component and use that, just loop through the array of products

Reply:
its just for preview i will be adding it soon